api/events/container.pb.go

```diff
@@ -202,127 +201,35 @@ func init() {
-	0xde, 0xf9, 0x17, 0x3c, 0xe5, 0x99, 0xfe, 0xac, 0xc0, 0xca, 0x08, 0x07, 0x6a, 0xff, 0x0f, 0x7e,
-	0x05, 0x00, 0x00, 0xff, 0xff, 0xf5, 0x09, 0xe0, 0xd6, 0x0b, 0x03, 0x00, 0x00,
+	// 402 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0x94, 0x52, 0xcd, 0x0a, 0xd3, 0x40,
+	0x18, 0xec, 0x26, 0xb5, 0xc5, 0x8d, 0xf8, 0xb3, 0xf4, 0x10, 0x73, 0x08, 0x6d, 0x4f, 0xc5, 0xc3,
+	0x06, 0xe2, 0x45, 0xed, 0x45, 0x6d, 0xf5, 0xa2, 0x82, 0x04, 0x84, 0xe2, 0x45, 0x36, 0xcd, 0x36,
+	0x5d, 0x4c, 0x76, 0x43, 0xb2, 0x29, 0xe4, 0xe6, 0xa3, 0xf8, 0x38, 0x3d, 0x78, 0xf0, 0xe8, 0x45,
+	0xb0, 0x79, 0x06, 0x1f, 0x40, 0xb2, 0x9b, 0xb4, 0xc1, 0x82, 0x3f, 0xa7, 0xcc, 0x0e, 0x33, 0xdf,
+	0x37, 0xf3, 0x11, 0xb8, 0x8c, 0x99, 0xdc, 0x97, 0x21, 0xde, 0x8a, 0xd4, 0xdb, 0x0a, 0x2e, 0x09,
+	0xe3, 0x34, 0x8f, 0xfa, 0x90, 0x64, 0xcc, 0xa3, 0x07, 0xca, 0x65, 0x71, 0x61, 0x71, 0x96, 0x0b,
+	0x29, 0xd0, 0xbd, 0x8b, 0x0c, 0x6b, 0x89, 0x73, 0x3f, 0x16, 0x22, 0x4e, 0xa8, 0xa7, 0x04, 0x61,
+	0xb9, 0xf3, 0x08, 0xaf, 0xb4, 0xda, 0x99, 0xc4, 0x22, 0x16, 0x0a, 0x7a, 0x0d, 0x6a, 0xd9, 0xa7,
+	0x7f, 0x0d, 0x70, 0x1e, 0x95, 0x25, 0x65, 0xcc, 0xb8, 0xb7, 0x63, 0x34, 0x89, 0x32, 0x22, 0xf7,
+	0x7a, 0xc2, 0xfc, 0x0b, 0x80, 0x77, 0x56, 0x9d, 0x7c, 0x95, 0x53, 0x22, 0x29, 0xba, 0x0d, 0x0d,
+	0x16, 0xd9, 0x60, 0x0a, 0x16, 0x37, 0x03, 0x83, 0x45, 0x68, 0x02, 0x6f, 0xb0, 0x94, 0xc4, 0xd4,
+	0x36, 0x14, 0xa5, 0x1f, 0x68, 0x0d, 0xc7, 0x79, 0xc9, 0x25, 0x4b, 0xa9, 0x6d, 0x4e, 0xc1, 0xc2,
+	0xf2, 0x1f, 0xe0, 0xab, 0x46, 0xf8, 0xb7, 0xd1, 0x38, 0xd0, 0x8e, 0xa0, 0xb3, 0x3a, 0x6f, 0xe0,
+	0xb8, 0xe5, 0x10, 0x82, 0x43, 0x4e, 0x52, 0xda, 0x2e, 0x56, 0x18, 0x61, 0x38, 0x16, 0x99, 0x64,
+	0x82, 0x17, 0x6a, 0xb9, 0xe5, 0x4f, 0xb0, 0xbe, 0x11, 0xee, 0x8a, 0xe1, 0x67, 0xbc, 0x0a, 0x3a,
+	0xd1, 0xfc, 0x7b, 0xbf, 0xce, 0xbb, 0x2c, 0xfa, 0xf7, 0x3a, 0x2f, 0xe1, 0x28, 0x21, 0x21, 0x4d,
+	0x0a, 0xdb, 0x9c, 0x9a, 0x0b, 0xcb, 0xc7, 0x7f, 0x6a, 0xa3, 0x27, 0xe3, 0xd7, 0xca, 0xf0, 0x82,
+	0xcb, 0xbc, 0x0a, 0x5a, 0x37, 0x9a, 0xc1, 0x5b, 0x05, 0x27, 0x59, 0xb1, 0x17, 0xf2, 0xc3, 0x47,
+	0x5a, 0xd9, 0x43, 0xb5, 0xc4, 0xea, 0xb8, 0x57, 0xb4, 0x72, 0x1e, 0x43, 0xab, 0xe7, 0x44, 0x77,
+	0xa1, 0xd9, 0x08, 0x75, 0xc0, 0x06, 0x36, 0x09, 0x0f, 0x24, 0x29, 0xcf, 0x09, 0xd5, 0xe3, 0x89,
+	0xf1, 0x08, 0xcc, 0x67, 0xbd, 0x7a, 0x6b, 0x9a, 0xd0, 0xeb, 0x7a, 0xcf, 0xdf, 0x1e, 0x4f, 0xee,
+	0xe0, 0xdb, 0xc9, 0x1d, 0x7c, 0xaa, 0x5d, 0x70, 0xac, 0x5d, 0xf0, 0xb5, 0x76, 0xc1, 0x8f, 0xda,
+	0x05, 0x9f, 0x7f, 0xba, 0xe0, 0xbd, 0xff, 0x1f, 0xbf, 0xec, 0x52, 0x7f, 0x36, 0x60, 0x63, 0x84,
+	0x23, 0x75, 0xef, 0x87, 0xbf, 0x02, 0x00, 0x00, 0xff, 0xff, 0x14, 0xa0, 0xd4, 0x8c, 0xf3, 0x02,
+	0x00, 0x00,
 }
 
-// Field returns the value for the given fieldpath as a string, if defined.
-// If the value is not defined, the second value will be false.
-func (m *ContainerCreate) Field(fieldpath []string) (string, bool) {
```

Comment:
Seems I need to implement this as a separate generator like https://github.com/containerd/ttrpc/pull/96.